<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vista</class>
 <widget class="QWidget" name="Vista">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vista</string>
  </property>
  <widget class="QLineEdit" name="leBuscar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="LineaDeTexto" name="lineaDeTexto">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>120</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePrimera">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSegunda">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAlta">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Dar de Alta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAgregar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineaDeTexto</class>
   <extends>QLineEdit</extends>
   <header>lineadetexto.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
